pkg/util/config/config.go

```diff
@@ -51,16 +51,42 @@ type Resources struct {
 	ShmVolume               *bool               `name:"enable_shm_volume" default:"true"`
 }
 
+type InfrastructureRole struct {
+	// Name of a secret which describes the role, and optionally name of a
+	// configmap with an extra information
+	SecretName spec.NamespacedName
+
+	Name     string
```

Comment:
Should this field better be called `User`? I got confused while configuring the `infrastructure_roles_secret` field in the config map (where I used `user:_<username>_` first, and then `name:user` and finally `name:_<username>_`).